### PR TITLE
create NSW campbelltown scraper

### LINKED ISSUE #51: [New Scraper] NSW: Campbelltown
**Council Name:**
Campbelltown Council

**Link to agenda webpage:**
https://www.campbelltown.nsw.gov.au/Council-and-Councillors/Meetings-and-Minutes

**Webpage quirks**
Once again business papers are grouped by year, and then each webpage lists the agenda and attachments like dot form.

**Additional context**
image

### PR DESCRIPTION
**Pull request by N-2-O**
resolves #51

### CODE REVIEW

**Comment by N-2-O** [comment]
> [Council Meetings](https://www.campbelltown.nsw.gov.au/Whats-On/Council-meetings) are held on the second Tuesday of the month at 6:30pm

Updated time and year

**Review by ArchangelEkim — CHANGES_REQUESTED** [review]
Hi Nathan, I noticed your code still uses some old imported; I'd advise ensuring you have pulled all the latest commits and reviewed the imports. 

Especially since the regexes can be imported too.

File: council_scrapers/scrapers/nsw/campbelltown.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import sys
+from pathlib import Path
+
+parent_dir = str(Path(__file__).resolve().parent.parent.parent)
+if parent_dir not in sys.path:
+    sys.path.append(parent_dir)
+
```
Comment (chrisobi02) [inline council_scrapers/scrapers/nsw/campbelltown.py:7]: leftover code from old file structure

File: council_scrapers/scrapers/nsw/campbelltown.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+from base_scraper import BaseScraper, register_scraper
+from logging.config import dictConfig
+from _dataclasses import ScraperReturn
```
Comment (chrisobi02) [inline council_scrapers/scrapers/nsw/campbelltown.py:10]: old code - dictConfig is unused and the base_scraper should be an absolute import (similarly _dataclasses should be removed)

File: council_scrapers/scrapers/nsw/campbelltown.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        webpage_url = "https://www.campbelltown.nsw.gov.au/Council-and-Councillors/Meetings-and-Minutes"
+        download_url = None
+
+        year = datetime.date.today().year
+        webpage_url += f"/{year}-Business-Papers#section-1"
+        output = self.fetch_with_requests(webpage_url)
+        soup = BeautifulSoup(output.content, "html.parser")
+
+        latest_meet = soup.find("h2")
+        pattern = r"([a-zA-Z\s]+)(\d{1,2}\s+(January|February|March|April|May|June|July|August|September|October|November|December))"
```
Comment (chrisobi02) [inline council_scrapers/scrapers/nsw/campbelltown.py:38]: compare to regex in constants.py

**Review by ArchangelEkim — APPROVED** [review]
It is updated to remove unnecessary code.

**Pull request merged**